frontstage/controllers/collection_instrument_controller.py

```diff
@@ -86,3 +87,12 @@ def upload_collection_instrument(upload_file, case_id, party_id):
         raise ApiError(logger, response)
 
     logger.info('Successfully uploaded collection instrument', case_id=case_id, party_id=party_id)
+
+
+def check_collection_instrument_size(upload_file, party_id, case_id):
+    logger.info('Checking the file size', party_id=party_id, case_id=case_id)
+    upload_file.save('/tmp/file')
```

Comment:
We shouldn't be saving anything to the file system.  You'll need to figure out another way to check for the size.